nntrainer/tensor/float_tensor.cpp

```diff
@@ -503,7 +503,8 @@ TensorV2 &FloatTensor::add(TensorV2 const &m, TensorV2 &output,
 }
 
 TensorV2 &FloatTensor::subtract(float const &value, TensorV2 &output) const {
-  throw std::logic_error("FloatTensor::subtract is not implemented yet");
+  auto f = std::bind(std::minus<float>(), std::placeholders::_1, value);
+  apply(f, output);
```

Comment:
Need to compare the performance. Inside of apply, we use std::transform. We need to check if it is faster than using SIMD Accelerator. If it is not, then it might be better to implement it.

Reply:
we are currently using std::transform for every main arithmetic operation. how should we compare with using a SIMD accelerator if there isn't any?